examples/integrations/neo4j/neo4j-se/pom.xml

```diff
@@ -104,6 +99,10 @@
                     <groupId>org.slf4j</groupId>
```

Comment:
Exclusions should be managed in Helidon. Is there any reason the exclusion should only be in the project that uses it?

Reply:
Fixed.